src/common/Cond.h

```diff
@@ -173,41 +173,41 @@ class C_SafeCond : public Context {
  * until wait() returns.
  */
 class C_SaferCond : public Context {
-  Mutex lock;    ///< Mutex to take
-  Cond cond;     ///< Cond to signal
-  bool done;     ///< true after finish() has been called
-  int rval;      ///< return value
+  ceph::mutex lock = ceph::make_mutex("C_SaferCond");    ///< Mutex to take
+  ceph::condition_variable cond;     ///< Cond to signal
+  bool done = false;     ///< true after finish() has been called
+  int rval = 0;      ///< return value
 public:
-  C_SaferCond() : lock("C_SaferCond"), done(false), rval(0) {}
-  explicit C_SaferCond(const std::string &name) : lock(name), done(false), rval(0) {}
+  C_SaferCond() = default;
+  explicit C_SaferCond(const std::string &name) : lock(ceph::make_mutex(name)), done(false), rval(0) {}
```

Comment:
Nit: `done` and `rval` are now default initialized above

Reply:
fixed!